person/src/com/person/utils/Validate.java

```diff
@@ -0,0 +1,21 @@
+package com.person.utils;
+
+public class Validate {
+    public static boolean name(String name, String rx)
+    {
+        if(name.length() > 255 || !name.matches(rx))
+            return false;
+
+        return true;
+    }
+
+    public static boolean gender(String gender)
+    {
+        return gender.equals("Male") || gender.equals("Female");
```

Comment:
@tatevikb you can get number instead of string from input and make gender as enum and do validation with get method

Reply:
@nsimonyan Since there were only 2 cases, I didn't do that way. OK, I'll update

Reply:
@nsimonyan OK, I moved name -> names.